src/components/subHeader/helpers/fetchRestaurantData.js

```diff
@@ -0,0 +1,12 @@
+// api
+import fetchData from "../../../api/fetch";
+
+// constants
+import baseUrl from "../../../constants/baseUrlForData";
+import { RESTAURANT_DATA_END_POINT } from "../constants/subheader.general";
+
+function fetchRestaurantData(restaurantId){
```

Comment:
Not sure though, but API calls are kinda generic for any application and if we have lot of endpoints then you would be having it for many different components in their general file and if tomorrow all endpoints changes we have to walkthrough each component and make those changes, if we keep the track at the same place inside api directory in some constant file then the changes can be made comfortably.